<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterValueWithTypeForm</class>
 <widget class="QWidget" name="ParameterValueWithTypeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="typeComboBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="parameterLabel">
     <property name="text">
      <string>Parameter:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="parameterComboBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="recommendationLabel">
     <property name="text">
      <string>Recommendation:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTextEdit" name="recommendationTextEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>typeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ParameterValueWithTypeForm</receiver>
   <slot>currentTypeIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>parameterComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ParameterValueWithTypeForm</receiver>
   <slot>currentIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
